src/vivarium/framework/components/manager.py

```diff
@@ -183,21 +208,46 @@ def add_components(self, components: Union[List, Tuple, Any]):
         """
         self._component_manager.add_components(components)
 
-    def get_components(self, component_type: Any) -> List:
+    def get_component(self, name: str) -> Any:
+        """Get the component that has ``name`` if presently held by the component
+        manager. Names are guaranteed to be unique.
+
+        Parameters
+        ----------
+        name
+            A component name.
+        Returns
+        -------
+            A component that has name ``name`` else None.
+
+        """
+        return self._component_manager.get_component(name)
+
+    def get_components_by_type(self, component_type: Any) -> List[Any]:
```

Comment:
Should this be ``Union[type, Tuple[type]]`` instead of ``Any``?

Reply:
yup just saw that in VPH